packages/cloud_functions/lib/cloud_functions.dart

```diff
@@ -29,16 +29,20 @@ class CloudFunctions {
   /// Executes this Callable HTTPS trigger asynchronously.
   ///
   /// @param functionName The name of the callable function being triggered.
+  /// @param region The region where the callable function being triggered. Default is us-central1.
   /// @param parameters Parameters to be passed to the callable function.
   Future<dynamic> call(
-      {@required String functionName, Map<String, dynamic> parameters}) async {
+      {@required String functionName,
+      String region,
```

Comment:
I think adding the default value in the parameters makes it more obvious, and then add an assert to the top of the function to guarantee that the region is not null and has been set.